src/app/[locale]/projects/components/ImportSBOMModal.tsx

```diff
@@ -96,27 +136,67 @@ const ImportSBOMModal = ({ importSBOMMetadata, setImportSBOMMetadata }: Props):
             const response = await ApiUtils.POST(
                 `projects/import/SBOM?type=${importSBOMMetadata.importType}`,
                 formData,
-                session.user.access_token
+                session.user.access_token,
             )
+            const responseText = await response.text()
```

Comment:
Can simply get responseJson
```suggestion
            const responseJson = await response.json()
```

Reply:
ResponseText is needed to display the error message on failure